src/app/users/view-user/view-user.component.ts

```diff
@@ -58,4 +59,24 @@ export class ViewUserComponent implements OnInit {
     });
   }
 
+
+  /**
+   * Change Password of the Users
+   */
+  changeUserPassword() {
+    const changeUserPasswordDialogRef = this.dialog.open(ChangePasswordDialogComponent, {
+      data: { password: `user ${this.userData.id}` }
```

Comment:
can you explain this line.

Reply:
passing data to the mat-dialog. But here it's not needed. I have removed that. I'm currently fixing the regex pattern